CODEOWNERS

```diff
@@ -51,4 +74,23 @@ platform/platform-paxweb-jettyconfig/*	@stustison
 
 platform/security/*		@stustison
 
+platform/security/security-migratable/*     @paouelle   @lambeaux
+
 platform/util/*			@stustison
+
+platform/migration/*            @paouelle   @lambeaux
+platform/platform-migratable/*  @paouelle   @lambeaux
+
+platform/platform-configuration/*       @lambeaux
+platform/platform-filter-clientinfo/*   @lambeaux
+
+
+###############################
+# ADMIN
```

Comment:
Na just change what there is a need for currently. *something something... preoptimization... something something* 😛